### ISSUE TITLE
Release a High Sierra version

### ISSUE THREAD

**Issue by bartkummel** [dustinrue/ControlPlane#498]
I see a "high sierra fixes" branch, which contains various fixes. I hope it is possible to create a bèta release out of this branch? At least one issue is causing me trouble on a daily basis, and it seems to be fixed on that branch: "fix crash when displaying default browser dialog". I would really appreciate a version of Control Plane with that fix applied, even if it is not perfect.

**Comment by AlexanderWillner** [comment]
Yes, generally a new release would be interesting @dustinrue - there are some changes since 2014..

**Comment by dustinrue** [comment]
Thank you for taking the time to report this. This issue was filed against the original Objective-C version of ControlPlane, which is now archived on the `master` branch.

The project has been **completely rewritten in Swift** — the new codebase is on the `main` branch and addresses many long-standing bugs including crashes on Apple Silicon, Ventura compatibility, the dark mode menu bar icon, Bluetooth stability, and more.

If you encounter this issue with the new Swift version, please open a fresh issue against `main`. Thanks again!
